frontend/app/helpers/spree/taxon_filters_helper.rb

```diff
@@ -0,0 +1,11 @@
+require 'spree/core/product_filters'
+
+module Spree
+  module TaxonFiltersHelper
+    def applicable_filters_for(_taxon)
+      [:brand_filter, :price_filter].map do |filter_name|
+        Spree::Core::ProductFilters.send(filter_name) if Spree::Core::ProductFilters.respond_to?(filter_name)
```

Comment:
I know this was copy-pasted from the existing implementation, but it's weird since both methods are already defined on `ProductFilters`. Expecting users to `remove_method` as the means of customization seems quite odd.

This of course is probably desirable for compatibility and shouldn't block this PR.